public/app/features/plugins/admin/components/PluginDetailsDisabledError.tsx

```diff
@@ -66,6 +66,19 @@ function renderDescriptionFromError(error?: PluginErrorCode): ReactElement {
           version of this plugin.
         </p>
       );
+    case PluginErrorCode.failedStart:
+      return (
+        <p>
+          Grafana failed to start this plugin and have therefore disabled it. Server logs can provide more information.
+        </p>
+      );
+    case PluginErrorCode.angular:
+      return (
+        <p>
+          This plugin is using AngularJS which is deprecated. Check if there is a new version of the plugin that updates
```

Comment:
Just want to check - does this _replace_ the Angular warning which would be displayed or would both be present?
<img width="998" alt="Screenshot 2024-04-16 at 10 14 38" src="https://github.com/grafana/grafana/assets/19860021/2d523f1a-9e85-4cd0-a73d-bbd12309f0a8">

I think we could just use the existing message for consistency?

Reply:
actually it now shows both:

![image](https://github.com/grafana/grafana/assets/4025665/29cfe41d-839f-44d5-9d3b-577beefd8b07)

I guess we can skip this additional warning since the explanation is already there.

Reply:
I have removed the duplicated message so now it looks like this (note that the new error is shown only if the plugin is installed):

![image](https://github.com/grafana/grafana/assets/4025665/6ca36de2-9e36-49ae-a936-4e6905e624c9)

Reply:
Ah I see, not the prettiest... is it possible to clear the angular warning if we display this error? Not sure if @xnyo has thoughts here

Reply:
Do we need to worry about looks for angular plugins? :smiley: I don't think many people visit this page, being just 8 plugins in the catalog still as angular plugins.

Reply:
all installed angular plugins would still be there though

Reply:
to clarify, installed plugins that are not in the catalog won't display this view (only the "plugin not found" screen). If you mean installed angular plugins that have a newer version which is not angular then yes, it will still show this message.